client/my-sites/marketplace/components/review-modal/index.tsx

```diff
@@ -25,12 +26,37 @@ export const ReviewModal = ( { isVisible, onClose, slug, productName, productTyp
 
 	const createReview = useCreateMarketplaceReviewMutation( { productType, slug } );
 
+	const handleSubmit = ( e: FormEvent ) => {
+		e.preventDefault();
+		createReview.mutate( {
+			productType,
+			slug: slug,
+			content: content,
+			rating: Number( rating ),
+		} );
+
+		recordTracksEvent( 'calypso_marketplace_reviews_add_submit', {
+			product_type: productType,
+			slug: slug,
+			rating: Number( rating ),
+		} );
+	};
+
+	const handleClose = () => {
+		recordTracksEvent( 'calypso_marketplace_reviews_add_dismiss', {
+			completed: createReview.isSuccess,
```

Comment:
Wouldn't it be better to use `is_success` instead of `completed`?

If you go within the logic of the `onComplete`, it should tell whether the operation happened, succeeding or not.

Reply:
Agreed e27ad7d0fc